playbooks/byo/openshift-checks/adhoc.yml

```diff
@@ -0,0 +1,16 @@
+---
+- name: OpenShift health checks
+  hosts: OSEv3
```

Comment:
With a patch to `logging.py` to return `{}`, what is not even a real check:

```
$ time ansible-playbook -i hosts playbooks/byo/openshift-checks/adhoc.yml -e checks=logging -vvv
...
PLAY RECAP ***************************************************************************************************************************************
localhost                  : ok=8    changed=0    unreachable=0    failed=0   
master1                    : ok=36   changed=2    unreachable=0    failed=0   
master2                    : ok=36   changed=2    unreachable=0    failed=0   
node1                      : ok=36   changed=2    unreachable=0    failed=0   
node2                      : ok=36   changed=2    unreachable=0    failed=0   


real    1m42.949s
user    0m36.358s
sys     0m30.438s
```

So that's more than a minute and a half to run the simplest of the checks :cry: 


But anyway, do you understand why the includes are in different files and why the tag always is only in one of them?